index.php

```diff
@@ -34,6 +34,12 @@ function searchedValue(): string {
     }
 }
 
+// Pagination
+$pageNumber = $_GET['pageNumber'] ?? 1;
+$numberOfDinosPerPage = 10;
+$offset = ($pageNumber-1) * $numberOfDinosPerPage;
+$totalPages = 10;
```

Comment:
Will this be a calculation in the next task rather than a set value?

Reply:
Yes